ee/modules/380-metallb/templates/speaker/daemonset.yaml

```diff
@@ -48,8 +48,11 @@ spec:
       {{- include "helm_lib_module_pod_security_context_run_as_user_root" . | nindent 6 }}
       nodeSelector:
         {{- .Values.metallb.speaker.nodeSelector | toYaml | nindent 8 }}
-{{- if .Values.metallb.speaker.tolerations }}
       tolerations:
+        - effect: NoSchedule
+          key: drbd.linbit.com/lost-quorum
+          operator: Exists
```

Comment:
Can't we reuse [`_helm_lib_additional_tolerations_storage_problems`](https://github.com/deckhouse/deckhouse/blob/51187d634eabe5c5b4c311fba7f348cbe3cf983f/helm_lib/charts/deckhouse_lib_helm/README.md#_helm_lib_additional_tolerations_storage_problems) in here?

https://github.com/deckhouse/deckhouse/blob/991cde1d374c3d049b41e6af93445ec5a394925c/helm_lib/charts/deckhouse_lib_helm/templates/_node_affinity.tpl#L226-L230

Reply:
done